packages/base/style/icon/chamfer.svg

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   width="14.3"
+   viewBox="0 0 21.45 14.3"
+   version="1.1"
+   height="14.3"
+   id="svg16"
+   sodipodi:docname="chamfer.svg"
+   inkscape:version="1.2.2 (b0a8486541, 2022-12-01)"
```

Comment:
Could you remove all the Inkscape metadata (like in this PR https://github.com/jupytercad/jupytercad/pull/338)? it will cause error messages when being used with React.